src/Compiler/Microsoft.AspNetCore.Razor.Language/src/Syntax/Generated/Syntax.xml.Internal.Generated.cs

```diff
@@ -4621,12 +4629,21 @@ public override GreenNode VisitRazorDirectiveBody(RazorDirectiveBodySyntax node)
 
   internal static partial class SyntaxFactory
   {
-    public static RazorDocumentSyntax RazorDocument(RazorBlockSyntax document)
+    public static RazorDocumentSyntax RazorDocument(RazorBlockSyntax document, SyntaxToken endOfFile)
     {
       if (document == null)
         throw new ArgumentNullException(nameof(document));
+      if (endOfFile == null)
+        throw new ArgumentNullException(nameof(endOfFile));
```

Comment:
My mind is blown by the fact that `SyntaxToken` is a class and can be null.

Reply:
Yeah, something on the list to change.